Applications/ApplicationsLib/LogogSetup.h

```diff
@@ -12,11 +12,28 @@
 
 #include <logog/include/logog.hpp>
 
+#if defined(USE_MPI)
+#include "BaseLib/LogogCustomCoutMPI.h"
+#include "BaseLib/LogogFormatterSuppressedMPI.h"
+	using LogogCout = BaseLib::LogogCustomCoutMPI;
+	using LogogFormat = BaseLib::LogogFormatterSuppressedMPI
+	                         <  TOPIC_GROUP_FLAG | TOPIC_FILE_NAME_FLAG
+	                          | TOPIC_LINE_NUMBER_FLAG>;
+#elif defined(USE_PETSC)
+#include "BaseLib/LogogCustomCoutPETSc.h"
+#include "BaseLib/LogogFormatterSuppressedPETSc.h"
+	using LogogCout = BaseLib::LogogCustomCoutPETSc;
+	using LogogFormat = BaseLib::LogogFormatterSuppressedPETSc
```

Comment:
With such template arguments, this format is equivalent to use  BaseLib::LogogSimpleFormatter.